src/cmake/build_libuhdr.cmake

```diff
@@ -0,0 +1,53 @@
+# Copyright Contributors to the OpenImageIO project.
+# SPDX-License-Identifier: Apache-2.0
+# https://github.com/AcademySoftwareFoundation/OpenImageIO
+
+######################################################################
+# libuhdr by hand!
+######################################################################
+
+set_cache (libuhdr_BUILD_VERSION 1.2.0 "libuhdr version for local builds")
+set (libuhdr_GIT_REPOSITORY "https://github.com/google/libultrahdr")
+set (libuhdr_GIT_TAG "v${libuhdr_BUILD_VERSION}")
+
+set_cache (libuhdr_BUILD_SHARED_LIBS OFF
+           DOC "Should execute a local libuhdr build, if necessary, build shared libraries" ADVANCED)
+
+find_package(JPEG REQUIRED)
+message("[libuhdr] JPEG_FOUND=${JPEG_FOUND}")
+message("[libuhdr] JPEG_INCLUDE_DIRS=${JPEG_INCLUDE_DIRS}")
+message("[libuhdr] JPEG_LIBRARIES=${JPEG_LIBRARIES}")
+message("[libuhdr] JPEG_VERSION=${JPEG_VERSION}")
```

Comment:
I think you shouldn't need this here at all, since in externalpackages.cmake `checked_find_package(libuhdr)` is called after the JPEG libraries are already found.

Reply:
Ah yeah sorry I'm just trying to debug the CI, I did not intend to leave that here !